lib/hop_swipe_cards.dart

```diff
@@ -130,23 +130,23 @@ class _HopSwipeCardsState extends State<HopSwipeCards>
   Alignment frontCardAlign = Alignment.center;
   Alignment afterSwipeAlignment = Alignment.center;
 
-  AnimationController _animationController;
+  AnimationController? _animationController;
 
-  int _currentFront;
+  int _currentFront = 0;
 
-  static TriggerDirection _trigger;
+  static TriggerDirection? _trigger;
 
   static bool triedSwipingLikeWhenNoPoints = false;
 
-  bool get isPointsLeft => widget._canSwipe == null || widget._canSwipe;
+  bool get isPointsLeft => widget._canSwipe == null || widget._canSwipe!;
```

Comment:
- You can remove `isPointsLeft`, and  alternate `isPointsLeft ` to `widget.isPointsLeft`

Reply:
@masashi-sutou 
Thank you for pointing out.
I fixed it